system_tests/monitoring.py

```diff
@@ -201,18 +202,23 @@ def test_write_point(self):
 
         retry_500(client.write_point)(metric, resource, VALUE)
 
-        def _query_timeseries_with_retries():
-            MAX_RETRIES = 7
+        MAX_RETRIES = 7
 
-            def _has_timeseries(result):
-                return len(list(result)) > 0
+        # need to wrap built-in function for decorators to work
+        def list_timeseries(query):
+            return list(query)
```

Comment:
It's in the block comment right above, functools.wrap is used by the decorator but requires that the function has a __module__ field, which the built-in functions don't have.